### ISSUE TITLE
Enable shared IP usage

### ISSUE THREAD

**Issue by misacek007** [teemtee/tmt#1134]
We have tests that require extra ip address to be available. That is IP address that is not assigned but can be reliably used  and is sure to not be taken by any third party.

An example of such tests are pacemaker resource agents binding the service (httpd, myslq) to this extra ip address on one of the multihost nodes or haproxy.

We need the possibility to specify the extra ip address in the specification of the test. 

This is not something that can be done by the test itself as it is different by environment where the test runs (different cloud providers have different ways of doing it, physical nodes often needs dhcp). 

The proposition of the format (2 extra ipv4 address requirements):
```
- ips: 2
```
As we use it in our beaker centric solution, this would end up with two extra ip addresses reserved in dhcp, periodically refreshed and written to /root/dhcp.leaseses file.

**Comment by happz** [comment]
Not necessarily, I think we did not think about this kind of requirement. The extra IPs do not have to be assigned, and in theory, it's perfectly fine to have just a single NIC and assign multiple IPs to it. What we have focused on NICs, we'd have to decouple IPs from them.
